README.md

```diff
@@ -40,29 +40,42 @@ At this time pya is more suitable for offline rendering than realtime.
 
 ## Authors and Contributors
 
-* Thomas Hermann, Ambient Intelligence Group, Faculty of Technology, Bielefeld University (author and maintainer)
-* Jiajun Yang, Ambient Intelligence Group, Faculty of Technology, Bielefeld University (co-author)
-* Alexander Neumann, Neurocognitions and Action - Biomechanics, Bielefeld University
+* [Thomas](https://github.com/thomas-hermann) (author, maintainer)
+* [Jiajun](https://github.com/wiccy46) (co-author, maintainer)
+* [Alexander](https://github.com/aleneum) (maintainer)
 * Contributors will be acknowledged here, contributions are welcome.
 
 ## Installation
 
-<!-- **Disclaimer**: We are currently making sure that pya can be uploaded to PyPI, until then clone the master branch and from inside the pya directory install via `pip install -e .` -->
+`pya` requires `portaudio` to play and record audio. 
 
-**Note**: pya can be installed using **pip**. But pya uses PyAudio for audio playback and record, and PyAudio 0.2.11 has yet to fully support Python 3.7. So using pip install with Python 3.7 may encounter issues such as portaudio. Solutions are:
+1. Disclaimer: As of December 2022, the latest `pyaudio==0.2.12` is not fully available on Conda Forge. This means 
```

Comment:
### Using Conda
Pyaudio can be installed via [conda](https://docs.conda.io):

```
conda install pyaudio
```

Disclaimer: Python 3.10+ requires PyAudio 0.2.12 which is not available on Conda as of December 2022. [Conda-forge](https://conda-forge.org/) provides a version only for Linux at the moment. Users of Python 3.10 should for now use other installation options.